core/src/test/scala/akka/persistence/cassandra/journal/CassandraJournalDeletionSpec.scala

```diff
@@ -46,10 +46,10 @@ object CassandraJournalDeletionSpec {
         }
       case GetRecoveredEvents =>
         sender() ! RecoveredEvents(recoveredEvents.reverse)
-      case DeleteTo(to) => deleteMessages(to)
+      case DeleteTo(to) =>
+        deleteMessages(to)
       case DeleteMessagesSuccess(to) =>
         context.system.log.debug("Deleted to: {}", to)
```

Comment:
this assert here has been removed as inside akka persistence delete confirmations to the PA are not ordered, which could be a very unimportant bug? I've confirmed that the journal does delete them in the correct order but hard to tests that explicitly